### DISCUSSION TITLE
项目中存在一些ts类型报错

### DISCUSSION THREAD

**Discussion by nieyuyao** [element-plus/element-plus#6234] [Q&A] [answered]
我拉取了仓库到本地，发现项目中存在很多ts类型报错，比如下面：

![image](https://user-images.githubusercontent.com/17698194/155252089-bfd2654e-c448-409c-80ff-2e4eacac6844.png)

是因为一些配置方面的问题么，还是本身就是有的？我已经执行了`pnpm i`并且`volar`等插件也是开启的。

**Comment by sxzz** [comment, marked as answer]
本身就有，我们正在逐步解决TS错误问题。
